### ISSUE TITLE
Feat: Others on the Floor (Single Item)

### ISSUE THREAD

**Issue by IkigaiLabsETH** [IkigaiLabsETH/ikigailabs.xyz#120]
![Image](https://github.com/IkigaiLabsETH/ikigailabs.xyz/assets/128307722/f8163a86-f183-4122-914a-252135ad6bec)

GM GM imagine seeing this GEM ON THE FLOOR on the homepage

I clicked on it an land on the single item page of this QQL mint pass

and now before I see see the footer I see a 2 rows of QQL nfts that are also on the floor but at much higher prices...

![Image](https://github.com/IkigaiLabsETH/ikigailabs.xyz/assets/128307722/c1908aac-fe2e-4998-8db9-9fedc4848f26)

that my dear fren would make me want to buy it even more

**Comment by royscheeren** [comment]
I'm not sure what you're asking here

**Comment by IkigaiLabsETH** [comment]
I would improve UX if on a single item NFT page, I see a row of Other NFTs from that collection check for example https://opensea.io/assets/ethereum/0x516d85f0c80d2c4809736aca3f3f95ce8545b5d2/110

![Screenshot 2024-06-28 at 19 01 33](https://github.com/IkigaiLabsETH/ikigailabs.xyz/assets/128307722/080ca5d9-8346-4717-95a9-5cf24ae0d9a5)

**Comment by royscheeren** [comment]
Ordered by price? Lowest first?

**Comment by IkigaiLabsETH** [comment]
yes because that will help a lot when I find a gem on the floor on homepage and I see others from that collection ordered by price, lowest first indeed
